### ISSUE TITLE
Nodes start OK despite failed DB migration

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#2997]

**Comment by at055612** [comment]
Now if another node fails the migration, then upon getting the bootstrap lock, this node will see
```
ERROR 2022-07-19T17:54:16.205Z main stroom.app.BootstrapUtil       Build version SNAPSHOT is marked as FAILED by node node1a. Either deploy a new version; or fix the issue, set column build_version.outcome to null then re-try this version. Aborting application startup. 
stroom.app.BootstrapUtil$BootstrapFailureException: Build version SNAPSHOT is marked as FAILED by node node1a. Either deploy a new version; or fix the issue, set column build_version.outcome to null then re-try this version. Aborting application startup.
	at stroom.app.BootstrapUtil.logFailureAndThrow(BootstrapUtil.java:309)
	at stroom.app.BootstrapUtil.lambda$doWithBootstrapLock$6(BootstrapUtil.java:218)
	at org.jooq.impl.DefaultDSLContext.lambda$transactionResult0$0(DefaultDSLContext.java:536)
	at org.jooq.impl.Tools$12$1.block(Tools.java:4843)
	at java.base/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3137)
	at org.jooq.impl.Tools$12.get(Tools.java:4840)
	at org.jooq.impl.DefaultDSLContext.transactionResult0(DefaultDSLContext.java:588)
	at org.jooq.impl.DefaultDSLContext.transactionResult(DefaultDSLContext.java:505)
	at stroom.app.BootstrapUtil.doWithBootstrapLock(BootstrapUtil.java:202)
	at stroom.app.BootstrapUtil.bootstrapApplication(BootstrapUtil.java:126)
	at stroom.app.BootstrapUtil.bootstrapApplication(BootstrapUtil.java:78)
	at stroom.app.App.run(App.java:196)
	at stroom.app.App.run(App.java:82)
	at io.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:43)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:87)
	at io.dropwizard.cli.Cli.run(Cli.java:78)
	at io.dropwizard.Application.run(Application.java:93)
	at stroom.app.App.main(App.java:136)

Process finished with exit code 1
```

**Comment by at055612** [comment]
Fixed in >7.0.0
